deploy-as-code/helm/environments/pb-prod-v2.yaml

```diff
@@ -74,7 +74,7 @@ cluster-configs:
               egov-enc-service: "http://egov-enc-service:8080/"
               egov-eis: "http://egov-eis:8080/"
               egov-workflow: "http://egov-workflow:8080/"
-              egov-workflow-v2: "http://egov-workflow-v2:8080/"
+              egov-workflow-v2: "http://egov-workflow-v2..egov:8080/"
```

Comment:
remove the double dot and without this how the workflow is working for PT and TL.